apis/projectcontour/v1/httpproxy.go

```diff
@@ -359,6 +581,12 @@ type VirtualHost struct {
 	// Only one of IPAllowFilterPolicy and IPDenyFilterPolicy can be defined.
 	// The rules defined here may be overridden in a Route.
 	IPDenyFilterPolicy []IPFilterPolicy `json:"ipDenyPolicy,omitempty"`
+
```

Comment:
can we condense this? 
left a comment on the design

Reply:
PTAL the updated design.